server/channels/app/export_test.go

```diff
@@ -1144,6 +1144,77 @@ func TestExportDeletedTeams(t *testing.T) {
 	}
 }
 
+func TestExportSingleTeam(t *testing.T) {
+	th1 := Setup(t).InitBasic()
+	defer th1.TearDown()
+
+	teams1, errTeams := th1.App.GetAllTeams()
+	assert.Nil(t, errTeams)
+	assert.Equal(t, len(teams1), 1)
+
+	channels1, err1 := th1.App.GetAllChannels(th1.Context, 0, 10, model.ChannelSearchOpts{})
+	assert.Nil(t, err1)
+	assert.Equal(t, 3, len(channels1))
+
+	// create DM and GM channels, which shouldn't be exported
+	dm1 := th1.CreateDmChannel(th1.BasicUser2)
+	th1.CreatePost(dm1)
+	user3 := th1.CreateUser()
+	gm1 := th1.CreateGroupChannel(th1.Context, th1.BasicUser2, user3)
+	th1.CreatePost(gm1)
+
+	// create additional team and channels outside of the team to export, to check they are excluded
+	team1 := th1.CreateTeam()
+	channel1 := th1.CreateChannel(th1.Context, team1)
+	th1.CreatePost(channel1)
+
+	all_teams, err1 := th1.App.GetAllTeams()
+	assert.Nil(t, err1)
+	assert.Equal(t, 2, len(all_teams))
+	all_channels, err1 := th1.App.GetAllChannels(th1.Context, 0, 10, model.ChannelSearchOpts{})
+	assert.Nil(t, err1)
+	assert.Equal(t, 6, len(all_channels))
+
+	var b bytes.Buffer
+	err := th1.App.BulkExport(th1.Context, &b, "somePath", nil, model.BulkExportOpts{TeamName: &th1.BasicTeam.Name})
+	require.Nil(t, err)
+
+	th2 := Setup(t)
+	defer th2.TearDown()
+	err, i := th2.App.BulkImport(th2.Context, &b, nil, false, 5)
+	assert.Nil(t, err)
+	assert.Equal(t, 0, i)
+
+	teams2, err := th2.App.GetAllTeams()
+	assert.Nil(t, err)
+	assert.Equal(t, 1, len(teams2))
+	assert.Equal(t, th1.BasicTeam.Name, teams2[0].Name)
+
+	channels2, err := th2.App.GetAllChannels(th1.Context, 0, 10, model.ChannelSearchOpts{})
+	assert.Nil(t, err)
```

Comment:
Per https://developers.mattermost.com/contribute/more-info/server/style-guide/#error-variable-names, `err` should be named `appErr`